src/include/optimizer/operators.h

```diff
@@ -147,13 +147,13 @@ class LogicalGroupBy : public OperatorNode<LogicalGroupBy> {
 class LogicalInsert : public OperatorNode<LogicalInsert> {
  public:
   static Operator make(
-      storage::DataTable *target_table, std::vector<char *> *columns,
-      std::vector<std::vector<peloton::expression::AbstractExpression *> *> *
+      storage::DataTable *target_table, std::vector<std::unique_ptr<char[]>> *columns,
```

Comment:
@hzxa21 @chenboy @luochenUmich Why do we have to use raw char arrays? Can we use `std::string` instead?

Reply:
We can use std::string. The constructor of InsertPlan uses vector<char*>*, so we use char* also in the logical operator. To make the codes be more C++, we should also avoid char* in the plan node.